mentat/session.py

```diff
@@ -100,6 +102,18 @@ def __init__(
         for path in paths:
             code_context.include(path, exclude_patterns=exclude_paths)
 
+    def _create_task(self, coro: Coroutine[None, None, Any]):
```

Comment:
This is the same as the one in the clients right? I wonder if it makes sense to make them inherit it from an abstract `TaskManager` class? Might make it more confusing rather than less though.

Reply:
If we have more than 2 places we need this functionality we can make a superclass haha